Recurly/BaseClient.cs

```diff
@@ -12,16 +15,18 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Serialization;
 using Newtonsoft.Json.Converters;
-using System.Net;
+using System.Threading;
+
+[assembly: InternalsVisibleTo("Recurly.Tests")]
 
 namespace Recurly {
     public class BaseClient {
         public string SiteId { get; }
         private string ApiKey { get; }
-        public IRestClient RestClient { get; set; }
+        private const string ApiUrl = "https://partner-api.recurly.com/";
         public virtual string ApiVersion { get; protected set; }
 
-        private const string API_URL = "https://partner-api.recurly.com/";
```

Comment:
Was the change from `API_URL` to `ApiUrl` just a style choice? What's the convention here?

Reply:
This is the recommended coding convention in stylecop: https://github.com/StyleCop/StyleCop